src/main/resources/config/application.yml

```diff
@@ -180,6 +180,8 @@ xm-config:
     kafka-config-topic: config_topic
 
 application:
+    jpa:
+        findOneByIdForUpdateTimeout: 10000
```

Comment:
I've suggestion to use common syntaxes, as I see we use kebab case - let's try to use kebab case everywhere: findOneByIdForUpdateTimeout -> find-one-by-id-for-update-timeout

Reply:
Already merged =(
I will fix it by separate pull request.